ppr-ui/src/components/mhrTransfers/LocationChange.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="mhr-location-change mt-8">
+    <FormCard
+      :label="content.sideLabel"
+      :showErrors="false"
+      :class="{'border-error-left': false}"
+    >
+      <template #formSlot>
+        <v-select
+          id="location-change-select"
+          v-model="locationChangeType"
+          :items="roleBasedLocationChangeTypes"
+          itemTitle="title"
+          itemValue="type"
+          variant="filled"
+          label="Location Change Type"
+          color="primary"
+        />
+      </template>
+    </FormCard>
+
+    <div
+      v-if="isTransportPermitType"
+      id="transport-permit-location-type"
+      class="pt-7"
+    >
+      [Transport Permit placeholder]
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, reactive, toRefs, watch } from 'vue'
+import { ContentIF } from '@/interfaces'
+import { locationChangeTypes } from '@/resources/mhr-transfers/transport-permits'
+import { FormCard } from '../common'
+import { LocationChangeTypes } from '@/enums'
+import { useStore } from '@/store/store'
+import { storeToRefs } from 'pinia'
+
+
+export default defineComponent({
+  name: 'LocationChange',
```

Comment:
Same as other comments, net new work like this was a good time to try, start and learn script setup syntax, which is why i started the other component in that manner. 
Little confused why this isn't following?

Reply:
Resolving this comment, as you address this above.